README.md

```diff
@@ -233,6 +238,17 @@ You may be interested in:
 * [demos/io_uring](./demos/io_uring), how ring buffers can eliminate kernel syscalls, reduce server hardware requirements by a factor of two, and change the way we think about event loops.
 * [demos/hash_table](./demos/hash_table), how linear probing compares with cuckoo probing, and what we look for in a hash table that needs to scale to millions (and billions) of account transfers.
 
+## Short-Term Roadmap
+
+Done:
+
+1. Harden the consensus layer
+
+In progress:
+
+2. Harden the storage layer (until this happens you'll occasionally see panics in the LSM code)
+3. Support querying an account for its transfers (until this happens you can only query account balances and list all transfers)
```

Comment:
We don't support listing all transfers. You can query accounts or transfers by id ("point queries").

Reply:
Whoops! Thanks